5Total.ipynb

```diff
@@ -0,0 +1,419 @@
+{
```

Comment:
<p>Here would be a good spot to check the actual data type size on your platform</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ubsuny/CP1_PythonCpp/pull/3/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>